alica_engine/src/engine/collections/SuccessMarks.cpp

```diff
@@ -143,19 +139,17 @@ bool SuccessMarks::succeeded(int64_t planId, int64_t entryPointId) const {
  * @param p An AbstractPlan*
  * @return A bool
  */
-bool SuccessMarks::anyTaskSucceeded(const AbstractPlan* p) const {
-    std::list<const EntryPoint*> l;
-    auto iter = this->successMarks.find(p);
-    if (iter != this->successMarks.end()) {
-        l = (*iter->second);
-        return (l.size() > 0);
+bool SuccessMarks::anyTaskSucceeded(const AbstractPlan* p) const
+{
+    auto iter = _successMarks.find(p);
+    if (iter != _successMarks.end()) {
+        return (!iter->second.empty() > 0);
```

Comment:
should be `return (!iter->second.empty());`

Reply:
nice find!